app/views/works/show.html.erb

```diff
@@ -0,0 +1,45 @@
+
+
+  <strong>
+
+    <h2> Title: <%= @work.title %></h2>
+    <p> Created by: <%= @work.creator %></p>
+    <p> Published: <%= @work.publication_year %></p>
+    <p> Description: <%= @work.description %></p>
```

Comment:
Why are these nested in a strong tag? If you wanted them to be bold, you shouldn't use HTML for that, you should use CSS!